connect_labs/audit/tests/test_rebuild_command_gate.py

```
@@ -0,0 +1,126 @@
+from connect_labs.audit.models import AuditSessionRecord
+from connect_labs.audit.prior_audit_models import PriorAuditProjectionState, PriorAuditVerdict
+from connect_labs.audit.prior_audit_projection import is_built
+
+OPP = 1973
+
+
+def _dt(day):
+    return datetime(2026, 5, day, tzinfo=timezone.utc)
+
+
+def _session(id, status, visit_results, completed_at=None, opportunity_id=OPP):
```

Ruff (0.16.2)

[error] Function argument `id` is shadowing a Python builtin

(A002)